Mapping Tools/views/Sliderator/SlideratorView.xaml.cs

```diff
@@ -70,7 +74,26 @@ public partial class SlideratorView {
             CanRun = false;
         }
 
-        private void SlideratorView_OnLoaded(object sender, RoutedEventArgs e) {
+        private void Import_Slider(object sender, RoutedEventArgs e)
+        {
+            bool editorRead = EditorReaderStuff.TryGetFullEditorReader(out var reader);
+                foreach (string path in MainWindow.AppWindow.GetCurrentMaps())
+                {
+                    var selected = new List<HitObject>();
+                    BeatmapEditor editor = editorRead ? EditorReaderStuff.GetNewestVersion(path, out selected, reader) : new BeatmapEditor(path);
+                    Beatmap beatmap = editor.Beatmap;
+                    Timing timing = beatmap.BeatmapTiming;
+                    List<HitObject> markedObjects = selected;
+                if (markedObjects.Count > 1 || markedObjects.Count < 1)
+                    continue;
+                if (!markedObjects.ToArray()[0].IsSlider)
```

Comment:
Why is this being cast to array. You can also get the first object from a list just fine.

Reply:
so change it to `markedObjects.FindAll(s => s.IsSlider).FirstOrDefault()`?

Reply:
You already ignore all counts above and below 1 so the markedObjects list has exactly 1 element. You can still just use !markedObjects[0].IsSlider

Reply:
Already resolved with latest commit